sources/packages/web/src/views/student/Login.vue

```diff
@@ -12,17 +12,14 @@
       >We are using BCSC for authentication. Please click on Login/Register
       buttons below to start your sign in/sign up.</v-card-text
     >
-    <v-card-actions>
+    <!-- TODO: v-btn inside v-card-actions is not working properly. try in vuetify 3 stable version -->
```

Comment:
The next PR for student BCeID will refactor this and it will no longer be a problem.